### PR TITLE
[DOCU-2315] Dev Portal section copy edits

### PR DESCRIPTION
**Pull request by Guaris**
### Summary
Enforces new style guidelines on Konnect section.

[docu-2315](https://konghq.atlassian.net/browse/DOCU-2315)
### Testing
Soon

### CODE REVIEW

File: app/konnect/dev-portal/customization/index.md [hunk 1]
```diff
@@ -52,36 +46,36 @@ From your domain registrar's DNS records settings options, point your CNAME to y
...
-2. Open the **Portal URL** tab.
+1. Open the **Portal URL** tab.
 
-3. Enter the full domain, including subdomain (if applicable). Don't include a path. It's not necessary to include the URL protocol, for example, `https://` into the **Custom Portal URL** field.
+2. Enter the full domain, including any applicable subdomain. 
+   
+   Don't include a path. It's not necessary to include the URL protocol, for example, `https://` into the **Custom Portal URL** field.
 
-4. Test to see if your custom URL works. You'll see the custom URL listed in your Dev Portal under your default Dev Portal URL. Your SSL certificate will be generated automatically.
+3. Test to see if the custom URL is accessible. You'll see the custom URL listed in the Dev Portal under the default Dev Portal URL. An SSL certificate will be generated automatically.
```
Comment (lena-larionova) [inline app/konnect/dev-portal/customization/index.md:57]: "Is accessible" implies accessibility in the ADA sense. 
```suggestion
3. Test your custom URL. You'll see the custom URL listed in the Dev Portal under the default Dev Portal URL. An SSL certificate will be generated automatically.
```

File: app/konnect/dev-portal/customization/index.md [hunk 2]
```diff
@@ -52,36 +46,36 @@ From your domain registrar's DNS records settings options, point your CNAME to y
...
 
    {:.note}
    > **Note:** DNS propagation can take a few hours. If after a few hours you can't access the Dev Portal from the custom URL, contact your domain registrar.
 
 ## Single Sign-On
 {:.badge .enterprise}
 
-A {{site.konnect_short_name}} Admin can configure Single Sign-On (SSO) for the Dev Portal via the Identity settings in the {{site.konnect_short_name}} Admin interface.
+A {{site.konnect_short_name}} admin can configure Single Sign-On (SSO) for the Dev Portal via the identity settings in the {{site.konnect_short_name}} admin interface.
```
Comment (lena-larionova) [inline app/konnect/dev-portal/customization/index.md:65]:
```suggestion
A {{site.konnect_short_name}} admin can configure single sign-on (SSO) for the Dev Portal via the identity settings in the {{site.konnect_short_name}} admin interface.
```

File: app/konnect/dev-portal/customization/index.md [hunk 3]
```diff
@@ -52,36 +46,36 @@ From your domain registrar's DNS records settings options, point your CNAME to y
...
 
-1. In {{site.konnect_short_name}}, open {% konnect_icon dev-portal %} **Dev Portal**, then click **Settings**.
+To configure single sign-on, Open {% konnect_icon dev-portal %} **Dev Portal**, click **Settings**, then follow these steps:
```
Comment (lena-larionova) [inline app/konnect/dev-portal/customization/index.md:67]:
```suggestion
To configure single sign-on, open {% konnect_icon dev-portal %} **Dev Portal**, click **Settings**, then follow these steps: 
```

File: app/konnect/dev-portal/customization/index.md [hunk 4]
```diff
@@ -52,36 +46,36 @@ From your domain registrar's DNS records settings options, point your CNAME to y
...
+A {{site.konnect_short_name}} admin can configure Single Sign-On (SSO) for the Dev Portal via the identity settings in the {{site.konnect_short_name}} admin interface.
 
-1. In {{site.konnect_short_name}}, open {% konnect_icon dev-portal %} **Dev Portal**, then click **Settings**.
+To configure single sign-on, Open {% konnect_icon dev-portal %} **Dev Portal**, click **Settings**, then follow these steps: 
 
-2. Open the **Identity** tab.
+1. Open the **Identity** tab.
 
    {:.note}
       > **Note:** Copy the callback URL and enter it in your OIDC Provider
```
Comment (lena-larionova) [inline app/konnect/dev-portal/customization/index.md:72]:
```suggestion
1. Copy the callback URL and enter it in your identity provider.
```
Not really sure why this is a note and not a step.

File: app/konnect/dev-portal/customization/index.md [hunk 5]
```diff
@@ -52,36 +46,36 @@ From your domain registrar's DNS records settings options, point your CNAME to y
...
 
-3. Enter the full domain, including the subdomain and protocol, into the  **Provider URL** field (also known as **Issuer**). For example,  `https://accounts.google.com` for Google IdP.
+2. Enter the full domain, including the subdomain and protocol, into the  **Provider URL** field (also known as **Issuer**). For example,  `https://accounts.google.com` for Google IdP.
```
Comment (lena-larionova) [inline app/konnect/dev-portal/customization/index.md:74]:
```suggestion
2. Enter the full domain, including the subdomain and protocol, into the  **Provider URL** field (also known as **Issuer**). For example, `https://accounts.google.com` for Google IdP.
```

File: app/konnect/dev-portal/dev-reg.md [hunk 6]
```diff
@@ -3,46 +3,49 @@ title: Sign Up for a Dev Portal Account
...
 
-All Developer registrations are either manually approved by {{site.konnect_short_name}} admins, or automatically approved by configuring your [auto-approve settings](/konnect/dev-portal/access-and-approval/auto-approve-devs-apps/).
+Developer registrations can be manually approved by {{site.konnect_short_name}} admins, or automatically approved by configuring your [auto-approve settings](/konnect/dev-portal/access-and-approval/auto-approve-devs-apps/).
 
 
 ## Register as a Developer {#register}
 
-All Developers must register through the {{site.konnect_short_name}} [Dev Portal](/konnect/servicehub/service-documentation/#publishing/). Because each Dev Portal has a unique URL, reach out to your {{site.konnect_short_name}} admin for the URL you should access.
+All Developers must register through the {{site.konnect_short_name}} [Dev Portal](/konnect/dev-portal/access/). A {{site.konnect_short_name}} admin can provide you with the correct registration URL.
```
Comment (lena-larionova) [inline app/konnect/dev-portal/dev-reg.md:13]:
```suggestion
All developers must register through the {{site.konnect_short_name}} [Dev Portal](/konnect/dev-portal/access/). A {{site.konnect_short_name}} admin can provide you with the correct registration URL. 
```

File: app/konnect/dev-portal/dev-reg.md [hunk 7]
```diff
@@ -3,46 +3,49 @@ title: Sign Up for a Dev Portal Account
...
+Developer registrations can be manually approved by {{site.konnect_short_name}} admins, or automatically approved by configuring your [auto-approve settings](/konnect/dev-portal/access-and-approval/auto-approve-devs-apps/).
 
 
 ## Register as a Developer {#register}
 
-All Developers must register through the {{site.konnect_short_name}} [Dev Portal](/konnect/servicehub/service-documentation/#publishing/). Because each Dev Portal has a unique URL, reach out to your {{site.konnect_short_name}} admin for the URL you should access.
+All Developers must register through the {{site.konnect_short_name}} [Dev Portal](/konnect/dev-portal/access/). A {{site.konnect_short_name}} admin can provide you with the correct registration URL. 
 
-1. Navigate to the {{site.konnect_short_name}} Dev Portal URL provided by your {{site.konnect_short_name}} admin.
+To register as a developer navigate to the {{site.konnect_short_name}} Dev Portal and follow these steps:
```
Comment (lena-larionova) [inline app/konnect/dev-portal/dev-reg.md:15]:
```suggestion
To register as a developer, navigate to the {{site.konnect_short_name}} Dev Portal and follow these steps: 
```

File: app/konnect/dev-portal/dev-reg.md [hunk 8]
```diff
@@ -3,46 +3,49 @@ title: Sign Up for a Dev Portal Account
...
+To register as a developer navigate to the {{site.konnect_short_name}} Dev Portal and follow these steps: 
 
-2. At the bottom of the login form, click **Sign Up**.
+1. Click **Sign Up**.
 
-3. Fill out the registration form and click **Create Account**.
+2. Fill out the registration form, then click **Create Account**.
 
-   You should receive an email to confirm your email address. Click the verification link in the email.
+3. A registration link will be sent to the email address you registered, click the verification link in the the registration email.
```
Comment (lena-larionova) [inline app/konnect/dev-portal/dev-reg.md:21]:
```suggestion
3. A registration link will be sent to the email address you registered. Click the verification link in the the registration email. 
```

File: app/konnect/dev-portal/dev-reg.md [hunk 9]
```diff
@@ -3,46 +3,49 @@ title: Sign Up for a Dev Portal Account
...
 
 ## Login/Register with SSO {#sso}
 {:.badge .enterprise}
-The {{site.konnect_short_name}} Dev Portal supports single sign-on (SSO) login and registration. The {{site.konnect_short_name}} Admin must configure SSO before it's available on the Dev Portal. For information on how to enable SSO on your Dev Portal as a {{site.konnect_short_name}} Admin, review the [enable single sign-on instructions](/konnect/dev-portal/customization/#single-sign-on/).
+The {{site.konnect_short_name}} Dev Portal supports single sign-on (SSO) login and registration. The {{site.konnect_short_name}} admin must configure SSO before it's available on the Dev Portal. For information on how to enable SSO on your Dev Portal as a {{site.konnect_short_name}} Admin, review the [enable single sign-on instructions](/konnect/dev-portal/customization/#single-sign-on/).
```
Comment (lena-larionova) [inline app/konnect/dev-portal/dev-reg.md:31]:
```suggestion
The {{site.konnect_short_name}} Dev Portal supports single sign-on (SSO) login and registration. The {{site.konnect_short_name}} admin must configure SSO before it's available on the Dev Portal. For information on how to enable SSO on your Dev Portal as a {{site.konnect_short_name}} admin, review the [single sign-on setup instructions](/konnect/dev-portal/customization/#single-sign-on/). 
```

File: app/konnect/dev-portal/dev-reg.md [hunk 10]
```diff
@@ -3,46 +3,49 @@ title: Sign Up for a Dev Portal Account
...
+If your admin has enabled auto-approve, you can log into the Dev Portal immediately after setting a password. For more information on auto-approving registration, read the [set up auto-approval for developer access as an admin](/konnect/dev-portal/access-and-approval/auto-approve-devs-apps/) documentation.
 
-If auto-approve is not enabled, an admin has to review and approve your request manually. All manually approved developers are notified via email upon gaining access to the Dev Portal. Learn how to [manually approve developer access as an admin](/konnect/dev-portal/access-and-approval/manage-devs/#approve-dev-reg).
+If auto-approve is not enabled, an admin has to review and approve your request manually. All manually approved developers are notified via email upon gaining access to the Dev Portal. To learn more about about the manual approval process read the [manually approve developer access as an admin](/konnect/dev-portal/access-and-approval/manage-devs/#approve-dev-reg) documentation.
```
Comment (lena-larionova) [inline app/konnect/dev-portal/dev-reg.md:27]:
```suggestion
If auto-approve is not enabled, an admin has to review and approve your request manually. All manually approved developers are notified via email upon gaining access to the Dev Portal. To learn more about about the manual approval process, read the [manual approval](/konnect/dev-portal/access-and-approval/manage-devs/#approve-dev-reg) documentation.
```

File: app/konnect/dev-portal/dev-reg.md [hunk 11]
```diff
@@ -3,46 +3,49 @@ title: Sign Up for a Dev Portal Account
...
 
 ## Login/Register with SSO {#sso}
 {:.badge .enterprise}
-The {{site.konnect_short_name}} Dev Portal supports single sign-on (SSO) login and registration. The {{site.konnect_short_name}} Admin must configure SSO before it's available on the Dev Portal. For information on how to enable SSO on your Dev Portal as a {{site.konnect_short_name}} Admin, review the [enable single sign-on instructions](/konnect/dev-portal/customization/#single-sign-on/).
+The {{site.konnect_short_name}} Dev Portal supports single sign-on (SSO) login and registration. The {{site.konnect_short_name}} admin must configure SSO before it's available on the Dev Portal. For information on how to enable SSO on your Dev Portal as a {{site.konnect_short_name}} Admin, review the [enable single sign-on instructions](/konnect/dev-portal/customization/#single-sign-on/). 
 
-1. Navigate to the {{site.konnect_short_name}} Dev Portal URL provided by your {{site.konnect_short_name}} admin.
+To login to the Dev Portal using SSO, navigate to the Dev Portal and follow these steps:
```
Comment (lena-larionova) [inline app/konnect/dev-portal/dev-reg.md:33]:
```suggestion
To log in to the Dev Portal using SSO, navigate to the Dev Portal and follow these steps: 
```

File: app/konnect/dev-portal/dev-reg.md [hunk 12]
```diff
@@ -3,46 +3,49 @@ title: Sign Up for a Dev Portal Account
...
-1. Navigate to the {{site.konnect_short_name}} Dev Portal URL provided by your {{site.konnect_short_name}} admin.
+To login to the Dev Portal using SSO, navigate to the Dev Portal and follow these steps: 
 
-2. From the login form, click **Login with SSO**.
+1. From the login form, click **Login with SSO**.
 
-3. After you are redirected to the Identity Provider (IdP) that the {{site.konnect_short_name}} Admin configured, sign in using your IdP's credentials.
-
-4. Once you've logged in, you are redirected to the {{site.konnect_short_name}} catalog.
+2. After you are redirected to the Identity Provider (IdP), log in using your IdP's credentials.
```
Comment (lena-larionova) [inline app/konnect/dev-portal/dev-reg.md:37]:
```suggestion
2. After you are redirected to the identity provider (IdP), log in using your IdP's credentials.
```

File: app/konnect/dev-portal/index.md [hunk 13]
```diff
@@ -20,7 +20,7 @@ The Dev Portal lives at a separate URL from {{site.konnect_short_name}} and requ
 
 ## Developer self service
 
-Dev Portals are created for Developers to use. The Dev Portal removes the burden of credential management from the API creator, and empowers developers to manage that aspect of the application development lifecycle on their own. When developers register their application to interact with services that have been exposed through Service Hub, they will have access to an overview page for the application that they registered, on this page, they can manage their API keys. Developers can also unregister for specific Services independently.
+Dev Portals are created for Developers to use. The Dev Portal removes the burden of credential management from the API creator, and empowers developers to manage that aspect of the application development lifecycle on their own. When developers register their application to interact with services that have been exposed through Service Hub, they will have access to an overview page for the application that they registered, on this page, they can manage their API keys. Developers can also unregister for specific services independently.
```
Comment (lena-larionova) [inline app/konnect/dev-portal/index.md:23]:
```suggestion
Dev Portals are created for developers to use. The Dev Portal removes the burden of credential management from the API creator, and empowers developers to manage that aspect of the application development lifecycle on their own. When developers register their application to interact with services that have been exposed through Service Hub, they will have access to an overview page for the application that they registered, on this page, they can manage their API keys. Developers can also unregister for specific services independently.
```

File: app/konnect/legacy/dev-portal/applications/enable-app-reg.md [hunk 14]
```diff
@@ -50,7 +50,7 @@ permissions.
     [Update Application](/konnect/legacy/dev-portal/applications/dev-apps#edit-my-app)
     dialog to match to your third-party OAuth2 claim.
 
-## Enable App Registration for the Key Authentication Flow {#konnect-key-auth-flow}
+## Enable App Registration for the Key Authentication Flow {#key-auth-flow}
```
Comment (lena-larionova) [inline app/konnect/legacy/dev-portal/applications/enable-app-reg.md:53]:
```suggestion
## Enable app registration for the key authentication flow {#key-auth-flow}
```

**Review by lena-larionova — CHANGES_REQUESTED** [review]

File: app/konnect/dev-portal/access-and-approval/manage-devs.md [hunk 15]
```diff
@@ -1,123 +1,101 @@
...
 
 {:.note}
-> **Note:** Rejected requests cannot be approved once rejected. This action is permanent.
+> **Note:** Rejecting a request is a permanent action. Rejected requests cannot be approved once rejected. 
 
 ### Delete access request {#delete-dev-reg}
 
-You can delete a developer's request to access your Dev Portal. This action cannot be undone. A developer
+You can delete a developer's access request. This action cannot be undone. A developer
 whose request was deleted will have to submit another request.
```
[the remark this answers is not in the document]
Comment (Guaris) [reply app/konnect/dev-portal/access-and-approval/manage-devs.md:57]: I'll find a way to reword this. I think the value here is expressing that even though a request is deleted a user can still submit another request. So if a user is abusing your service by requesting too many applications, simply deleting the application will not solve the problem, and you have to actually remove the user.

File: app/konnect/dev-portal/access-and-approval/manage-app-reg-requests.md [hunk 16]
```diff
@@ -1,100 +1,79 @@
...
-   <!---An email is sent to the developer to let them know their application registration
-   for Service access was rejected.--->
 
 
-### Delete an app registration request {#delete-app-reg}
+## Delete an app registration request {#delete-app-reg}
 
-After a request is deleted, a developer can register the app for the Service again. The application still
-appears in the Select Application list in the Register for Service dialog for the Service. The Delete action cannot be undone.  
+After a request is deleted, a developer can re-submit an application registration request. The Delete action cannot be undone.
```
[the remark this answers is not in the document]
Comment (Guaris) [reply app/konnect/dev-portal/access-and-approval/manage-app-reg-requests.md:50]: I don't think so. Dev-reg-app-service is information for a developer who is using the developer portal to register a service. This information informs the konnect admin that deleting a request does not prevent new ones from being made.

File: app/konnect/dev-portal/applications/enable-app-reg.md [hunk 17]
```diff
@@ -62,43 +62,41 @@ Service. Now, click the **Versions** button and select the desired version.
...
+   |:---------------|:----------------------------------------------------------------------------------|--|
+   | `Issuer` | The issuer URL from which the OpenID Connect configuration can be discovered. For example: `https://dev-1234567.okta.com/oauth2/default`.  |**True** |
+   | `Scopes` | The scopes to be requested from the OpenID Provider. Enter one or more scopes separated by spaces, for example: `open_id` `myscope1`.  | **False**
+   | `Consumer claims` |  Name of the claim that is used to find a consumer. | **True**
+   | `Auth method` | The supported authentication method(s) you want to enable. This field should contain only the authentication methods that you need to use. Individual entries must be separated by commas. Available options: `password`, `client_credentials`, `authorization_code`, `bearer`, `introspection`, `kong_oauth2`, `refresh_token`, `session`. | **True**
+   | `Hide Credentials` |**Default: disabled** Hide the credential from the upstream service. If enabled, the plugin strips the credential from the request header, query string, or request body, before proxying it. | **False** |
```
[the remark this answers is not in the document]
Comment (Guaris) [reply app/konnect/dev-portal/applications/enable-app-reg.md:78]: I like the way it looks in your suggestion more so than in the first column.

**Comment by Guaris** [comment]
@lena-larionova let me know what you think about my solution to your comment about a developer being able to re-reg. If it still feels off I'll delete it. These two files: 
app/konnect/dev-portal/access-and-approval/manage-app-reg-requests.md
app/konnect/dev-portal/access-and-approval/manage-devs.md

**Pull request merged**